ntoskrnl/io/pnpmgr/pnpreport.c

```diff
@@ -294,6 +294,15 @@ IoReportDetectedDevice(
         return Status;
     }
 
+    /* Store the value for device description (required by user-mode New device installer) */
+    Status = IopInitializeDeviceDescription(DeviceNode, InstanceKey);
+    if (!NT_SUCCESS(Status))
+    {
+        DPRINT("Failed to write the device description: 0x%x\n", Status);
```

Comment:
```suggestion
        DPRINT1("Failed to write the device description: 0x%x\n", Status);
```

Reply:
This was done the same way as debug prints above added code.

Reply:
Right, yet I would think all these failures should be `DPRINT1()`.